src/modules/sub-items/components/view-switcher/view.switcher.component.js

```diff
@@ -67,8 +67,13 @@ export default class ViewSwitcherComponent extends Component {
             componentClassName = `${componentClassName} ${componentClassName}--disabled`
         }
 
+        const attrs = {
+            className: componentClassName,
+            title: this.props.tooltipLabel
```

Comment:
I think it would be better to have a tooltip on each button ('List view', 'Grid view') instead of one 'Change view'.